### DISCUSSION TITLE
get_users not working

### DISCUSSION THREAD

**Discussion by Wittline** [tweepy/tweepy#1714] [Questions] [answered]
The below line is throwing an error:

users = api.get_users(ids = ['11111010109','193764933'])

error:

**users = api.get_users(ids = ['11111010109','193764933'])
AttributeError: 'API' object has no attribute 'get_users'**

Is anyone else facing the same error?

**Comment by Harmon758** [comment, marked as answer]
Like the error says, [`API`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API) doesn't have a `get_users` attribute/method.
You probably want to use [`API.lookup_users`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API.lookup_users).

For code block usage, see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.

**Reply by Wittline** [reply]
"**lookup_user**" does not bring me enough information about the users in the response, I just tried with the **client** object and i am still receiving the same error.

**Reply by Harmon758** [reply]
`lookup_user` is not an [`API`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API) or [`Client`](https://tweepy.readthedocs.io/en/v4.4.0/client.html#tweepy.Client) method either.
Like the documentation says, [`API.lookup_users`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API.lookup_users) returns fully-hydrated user objects.
[`Client.get_users`](https://tweepy.readthedocs.io/en/v4.4.0/client.html#tweepy.Client.get_users) does exist. What's the full traceback?

**Reply by Wittline** [reply]
working now, through the **client** object.

```python
users = client.get_users(ids = ['1331924150'], user_fields= ['created_at', 'description', 'id', 'location', 'name',  'profile_image_url', 'protected', 'public_metrics', 'url', 'username', 'verified'])
print(users[0][0].id)
```
